pkg/operator/ceph/csi/template/nfs/csi-nfsplugin-provisioner-dep.yaml

```diff
@@ -21,6 +21,27 @@ spec:
       priorityClassName: {{ .ProvisionerPriorityClassName }}
       {{ end }}
       containers:
+        {{ if .NFSAttachRequired }}
+        - name: csi-attacher
+          image: {{ .AttacherImage }}
+          args:
+            - "--v={{ .SidecarLogLevel }}"
+            - "--csi-address=$(ADDRESS)"
+            - "--leader-election=true"
+            - "--timeout={{ .GRPCTimeout }}"
+            - "--leader-election-namespace={{ .Namespace }}"
+            - "--leader-election-lease-duration=137s"
+            - "--leader-election-renew-deadline=107s"
+            - "--leader-election-retry-period=26s"
+            - "--default-fstype=ext4"
```

Comment:
nfs  need to config filesystem?

Reply:
This will not be used we can remove it for both cephfs and nfs. I have taken the copy from the cephfs deployment template.